docs/developer/developer-handbook.md

```diff
@@ -298,21 +307,243 @@ found [here](https://www.w3.org/TR/json-ld11/#the-context).
 
 #### Policies
 
+Policies are the EDC way of expressing that certain conditions may, must or must not be satisfied in certain situations.
+In that sense, policies are used to express what requirements a subject (e.g. a communication partner) must satisfy in
+order to be able to perform an action. For example, one such requirement - or more specifically: a _duty_ could be that
+a communication partner who wants to negotiate a contract, must have headquarters in the European Union.
+
+Policies are [ODRL](https://www.w3.org/TR/odrl-model/) serialized as JSON-LD. Thus, our previous example would look like
+this:
+
+```json
+{
+  "@context": {
+    "edc": "https://w3id.org/edc/v0.0.1/ns/"
+  },
+  "@type": "PolicyDefinition",
+  "policy": {
+    "@context": "http://www.w3.org/ns/odrl.jsonld",
+    "@type": "Set",
+    "duty": [
+      {
+        "target": "http://example.com/asset:12345",
+        "action": "use",
+        "constraint": {
+          "leftOperand": "headquarter_location",
+          "operator": "eq",
```

Comment:
Probably we should mention that `leftOperand` value is subject to expansion cause the `odrl`  type is `@vocab` and in this case it will be expanded as `https://w3id.org/edc/v0.0.1/ns/headquarter_location` as the default `@vocab` is the edc one. If users don't want this namespace they should bring their own.

Reply:
wouldn't this also be the case for the operator and rightOperand?

Reply:
the `rightOperand` value is ok it doesn't get touched on the expansion process,

the operand value will be expanded automatically to `odrl` namespace i think